ts/Stock/Indicators/Momentum/MomentumIndicator.ts

```diff
@@ -67,11 +67,8 @@ class MomentumIndicator extends SMAIndicator {
      * @requires     stock/indicators/momentum
      * @optionparent plotOptions.momentum
      */
-    public static defaultOptions: MomentumOptions = merge(SMAIndicator.defaultOptions, {
-        params: {
-            period: 14
-        }
-    } as MomentumOptions);
+    public static defaultOptions: MomentumOptions =
+    merge(SMAIndicator.defaultOptions, {} as MomentumOptions);
```

Comment:
Is th `merge` function needed here? 
couldn't it be just 
`public static defaultOptions = SMAIndicator.defaultOptions as MomentumOptions`

Reply:
Unfortunately, it generates an error ```code tagged with @optionparent must be an object``` when generating API. I'm leaving it as it is.

Reply:
FYI @bre1470

Reply:
This is correct, it creates a clone and also follows the pattern for JSDoc.

Reply:
Anyway, it's resolved now, because I'm not merging an empty object anymore.